src/wx/panel.cpp

```diff
@@ -1476,7 +1476,7 @@ class FilterThread : public wxThread {
     // if NULL, end thread
     uint8_t* src;
 
-    ExitCode Entry()
+    int Entry()
```

Comment:
Any ideas what the right fix for this one would be?

I assumed the linter was complaining because the type of `ExitCode` doesnt match the integers that are being returned in the function body.